### PR TITLE
Add deployment instructions

### PR DESCRIPTION
**Pull request by mxie**
Fixes #20.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,6 +61,20 @@ Use a `locale` or `l` parameter: `http://docs.railsbridge.org/?l=es`.
 
 Note that in this mode, links are not rewritten, so if they fail you will have to manually add the parameter again.
 
+## Deploying
+
+Only organizers and invited collaborators are allowed to deploy the app.
+
+To deploy, you must have a remote branch set up that points to the Heroku app.
```
Comment (gabebw) [inline README.md]: I think of it as adding a remote, not a remote branch. How about:

> To deploy, you must first add the Heroku remote:

Comment (gabebw) [inline README.md]: Or, what about adding a bin/setup script?

Comment (mxie) [inline README.md]: You're right. This was a weird one to phrase. Fixing!

Comment (mxie) [inline README.md]: I think I'd like to do that (the setup script) in a separate iteration. There are a few other setup-related things in the README I'd like to change.

**Comment by gabebw** [comment]
:+1:

**Pull request merged**